build/musicbrainz-dev/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM metabrainz/base-image
+LABEL Maintainer="yvanzo@metabrainz.org"
+
+ENV DOCKERIZE_VERSION v0.5.0
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
+    tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
+    rm -f dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+ARG POSTGRES_VERSION=9.5
+RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+    echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    apt-key adv --keyserver keyserver.ubuntu.com --refresh-keys 'Yarn' && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    curl -sLO https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
+    curl -sLO https://deb.nodesource.com/node_10.x/pool/main/n/nodejs/nodejs_10.15.0-1nodesource1_amd64.deb && \
+    apt-get update && \
+    apt-get install \
+        --no-install-recommends \
+        --no-install-suggests \
+        -y \
+        ./google-chrome-stable_current_amd64.deb \
+        ./nodejs_10.15.0-1nodesource1_amd64.deb \
+        cpanminus \
+        bash-completion \
+        build-essential \
+        gettext \
+        git-core \
+        language-pack-de \
+        language-pack-el \
+        language-pack-es \
+        language-pack-et \
+        language-pack-fi \
+        language-pack-fr \
+        language-pack-it \
+        language-pack-ja \
+        language-pack-nl \
+        language-pack-sq \
+        libdb-dev \
+        libexpat1-dev \
+        libicu-dev \
+        liblocal-lib-perl \
+        libpq-dev \
+        libxml2-dev \
+        libssl-dev \
+        memcached \
+        postgresql-${POSTGRES_VERSION} \
+        python-minimal \
+        wget \
+        yarn \
+        zlib1g-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY DBDefs.pm /
+COPY scripts/* /
+RUN cat /snippet.perllocallib.bashrc >> ~/.bashrc \
+    && rm /snippet.perllocallib.bashrc
+
+# Postgres user/password would be solely needed to compile tests
+ARG POSTGRES_USER=doesntmatteraslongasyoudontcompiletests
+ARG POSTGRES_PASSWORD=doesntmatteraslongasyoudontcompiletests
+
+ENV MUSICBRAINZ_CATALYST_DEBUG=0 \
+    MUSICBRAINZ_DEVELOPMENT_SERVER=1 \
+    MUSICBRAINZ_STANDALONE_SERVER=1 \
+    MUSICBRAINZ_WEB_SERVER_HOST=localhost \
+    MUSICBRAINZ_WEB_SERVER_PORT=5000 \
+    POSTGRES_USER=musicbrainz \
+    POSTGRES_PASSWORD=musicbrainz
+
+VOLUME  ["/media/dbdump"]
+VOLUME  ["/musicbrainz-server"]
```

Comment:
I am not a big fan of putting volumes in docker images as it force creates them and there is not way to turn them off. Putting them in the docker-compose.yml should be enough.

Reply:
Seems sensible, more details: https://boxboat.com/2017/01/23/volumes-and-dockerfiles-dont-mix/

We should probable remove `/media/dbdump` from `build/musicbrainz/Dockerfile` too.

Reply:
Reverted adding `VOLUME` for `/musicbrainz-server` in https://github.com/metabrainz/musicbrainz-docker/pull/118/commits/771c75edd69039ae7ed3516d0a1a64d310c987b0 (pending rebase).

Reply:
Removed `/media/dbdump` from Dockerfile in 33dc6b3393392fbcfd8a435ae66c6d22d87a58f3 on `mbvm-38-dev` branch.

Reply:
Rebased and removed both `VOLUME` instructions.